igvm/commands.py

```diff
@@ -432,6 +432,10 @@ def _reset_hypervisor():
                 _vm.start(transaction=transaction)
             _vm.reset_state()
 
+            # Add migration log entries to source and destination hypervisor
```

Comment:
No need for a change, but maybe change the comment to swap source and destination or even better swap previous_hypervisor and hypervisor - which would sound more logical to me.